src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/CalculationConfig.jsx

```diff
@@ -37,6 +37,21 @@ const CalculationConfig = (props) => {
   const { umap: umapSettings, tsne: tsneSettings } = data?.embeddingSettings.methodSettings || {};
   const { louvain: louvainSettings } = data?.clusteringSettings.methodSettings || {};
 
+  const [resolution, setResolution] = useState(null);
+  const [minDistance, setMinDistance] = useState(null);
```

Comment:
Introduced local states as "buffer" before firing dispatch to redux when state is ultimately set.